src/reducers/license-change.js

```diff
@@ -4,6 +4,9 @@ const defaultState = {
     const pathArr = urlPath.split('/');
     // the license should be the first item (not counting the 0 index which is an empty string)
     // replace it with the new id
+    if (pathArr[0] === '') {
+      pathArr.shift();
```

Comment:
A little bit concerned that we are editing the array here, as `pathArr[1]` will now be pointing to a different array element to what it would have before this change.

What is the reason behind this change?

Reply:
This is a newly created array, that just takes urlPath, and splits it by '/'; as it might start with '/' - sometimes we get empty string as the first element. With this check we remove it from array if that's the case.